embrace-android-fcm/src/main/java/io/embrace/android/embracesdk/fcm/swazzle/callback/com/android/fcm/FirebaseSwazzledHooks.java

```diff
@@ -100,10 +97,17 @@ private static void handleRemoteMessage(@NonNull RemoteMessage message) {
                         hasData
                 );
             } catch (Exception e) {
-                logger.logError("Failed to log push Notification", e);
+                logError("Failed to log push Notification", e);
             }
         } catch (Exception e) {
-            logger.logError("Push Notification Error", e);
+            logError("Push Notification Error", e);
+        }
+    }
+
+    private static void logError(@NonNull String message, @Nullable Exception e) {
+        Embrace.getInstance().getInternalInterface().logError(message, null, null, false);
```

Comment:
nit: extract `getInternalInterface` to variable

Reply:
I'm going to leave this as is for now. I like idea of making internal error recording more explicit and I'll do this as part of that refactor.